domain/test/main.test.js

```diff
@@ -78,6 +78,42 @@ describe('Тест для кладовки', function () {
         assert(worst.dps === item4.dps);
     });
 
+    it('should be fully to update character', async function () {
+        let char = { dmg: 100, attackSpd: 10 };
+        let id = await klad.placeInKladovka(db, coll, char);
+        char.dmg = 200;
+        char.attackSpd = 20;
+        let up = await klad.updateFullyInKladovka(db, coll, id, char);
+        assert(up === 1);
+        let res = await klad.getFromKladovka(db, coll, id);
+        assert(res.dmg === 200);
+    });
+
+    it('should be error at update character with not existsed id', async function () {
```

Comment:
"existsed"? Again? Where do you get that extra "s" all the time? What about that spell checker? We had troubles setting it up for russian, but there are plenty of spell checkers for english.
And fix title for test.